ecs_pod_creator.go

```diff
@@ -126,16 +139,37 @@ func (o *ECSPodCreationOptions) SetExecutionOptions(opts ECSPodExecutionOptions)
 	return o
 }
 
-// validateContainerDefinitions checks that all the individual container definitions are valid.
+// validateContainerDefinitions checks that all the individual container
+// definitions are valid.
 func (o *ECSPodCreationOptions) validateContainerDefinitions() error {
-	var totalContainerMemMB, totalContainerCPU int
 	catcher := grip.NewBasicCatcher()
 
 	catcher.NewWhen(len(o.ContainerDefinitions) == 0, "must specify at least one container definition")
 
+	var networkMode ECSNetworkMode
+	if o.NetworkMode != nil {
+		networkMode = *o.NetworkMode
+	} else {
+		networkMode = NetworkModeBridge
```

Comment:
Should this check if the pod is running on windows and set it to use the windows NAT network mode? I also see that this mode doesn't exist.

Reply:
There's no way to test that the container is a Windows container. NAT is the implicit default value for Windows. The only other network mode that you're allowed to specify is "awsvpc".